kafka-ui-api/src/main/java/com/provectus/kafka/ui/model/InternalConsumerGroup.java

```diff
@@ -80,4 +72,29 @@ private static Long calculateMessagesBehind(Map<TopicPartition, Long> offsets, M
     return messagesBehind;
   }
 
+  private static Integer calculateTopicNum(Map<TopicPartition, Long> offsets, Collection<InternalMember> members) {
+
+    return Stream.concat(
+        offsets.keySet().stream().map(TopicPartition::topic),
+        members.stream()
+            .flatMap(m -> m.getAssignment().stream().map(TopicPartition::topic))
+    ).collect(Collectors.toSet()).size();
```

Comment:
Please use `.distinct().count()` instead

Reply:
Done. 
In order to have compatibility to the topic field type in consumerGroupDTO I casted the resulted long to Integer. is it fine ?